### DISCUSSION TITLE
Wheel slippage question

### DISCUSSION THREAD

**Discussion by eeeewwqq** [JSBSim-Team/jsbsim#807] [Q&A] [answered]
Hi! I'm developing game and I wonder if something can be used for drawing tire tracks on landing.
For example when landing on huge speed with brakes enabled I think there will be some slippage.

At what props to take look to implement it?

**Comment by dpculp** [comment, marked as answer]
A simple way would be to use `gear/unit[1]/WOW` , which is "weight on wheels", and has a value of zero while airborne and changes to 1.0 when that gear strut starts to compress.  You could begin the wheel mark at that point and extend it some predefined length.

If you want to get fancy you could also use the property `gear/unit[1]/wheel-speed-fps` and compare it to the ground speed of the airplane in fps.  If the WOW is true and the wheel speed differs from the airplane ground speed by some value, then draw the mark.

Gear unit 1 is *usually* the left main gear, and gear unit 2 is *usually* the right main gear, but this may not always be the case since the FDM author has freedom to number the gear any way he wants.

-- Dave

**Comment by eeeewwqq** [comment]
> If you want to get fancy you could also use the property gear/unit[1]/wheel-speed-fps and compare it to the ground speed of the airplane in fps. If the WOW is true and the wheel speed differs from the airplane ground speed by some value, then draw the mark.

Thanks, exactly what I was looking for
